mypy_django_plugin/plugins/models.py

```diff
@@ -200,13 +236,11 @@ def extract_ref_to_fullname(rvalue_expr: CallExpr,
 def process_model_class(ctx: ClassDefContext) -> None:
     initializers = [
         InjectAnyAsBaseForNestedMeta,
-        AddDefaultObjectsManager,
+        AddManagerTypes,
+        AddDefaultManager,
         AddIdAttributeIfPrimaryKeyTrueIsNotSet,
         SetIdAttrsForRelatedFields,
         AddRelatedManagers
     ]
     for initializer_cls in initializers:
-        initializer_cls.from_ctx(ctx).run()
-
-    # allow unspecified attributes for now
-    ctx.cls.info.fallback_to_any = True
```

Comment:
Leave it be. There was some problems (I'll give you an example later, if you want) with cyclical dependencies in mypy, which prevent from filling model classes with all the attributes now. 
In 0.680, or maybe in 0.690, when mypy guys will finish semanal refactoring, they should be alleviated, and we'll remove this line. For now, it's just a bunch of false positives.